float32 prev_lat
float32 prev_lon
float32 current_lat 
float32 current_lon
---
float32 gps_heading

